--- a/sysdeps/x86_64/Makefile
+++ b/sysdeps/x86_64/Makefile
@@ -209,15 +209,6 @@ LDFLAGS-tst-plt-rewritemod2.so = -Wl,-z,now,-z,undefs
 tst-plt-rewrite2-ENV = GLIBC_TUNABLES=glibc.cpu.plt_rewrite=2
 $(objpfx)tst-plt-rewrite2: $(objpfx)tst-plt-rewritemod2.so
 
-tests-special += $(objpfx)check-dt-x86-64-plt.out
-
-$(objpfx)check-dt-x86-64-plt.out: $(common-objpfx)libc.so
-	LC_ALL=C $(READELF) -V -W $< \
-		| sed -ne '/.gnu.version_d/, /.gnu.version_r/ p' \
-		| grep GLIBC_ABI_DT_X86_64_PLT > $@; \
-	$(evaluate-test)
-generated += check-dt-x86-64-plt.out
-
 tests-special += $(objpfx)check-gnu2-tls.out
 
 $(objpfx)check-gnu2-tls.out: $(common-objpfx)libc.so
--- a/sysdeps/x86_64/Versions
+++ b/sysdeps/x86_64/Versions
@@ -10,11 +10,6 @@ libc {
     # by scripts/versions.awk.
     __placeholder_only_for_empty_version_map;
   }
-  GLIBC_ABI_DT_X86_64_PLT {
-    # This symbol is used only for empty version map and will be removed
-    # by scripts/versions.awk.
-    __placeholder_only_for_empty_version_map;
-  }
 }
 libm {
   GLIBC_2.1 {
