commit 559010e471acb3cb292615b71b248aba73e5c2fe
Author: Dragan Stanojević (Nevidljivi) <neville.ravenwood@gmail.com>
Date:   Wed Feb 7 16:31:04 2024 +0100

    localedata: hr_HR: change currency to EUR/€
    
    Resolves: BZ # 29845

diff --git a/localedata/locales/hr_HR b/localedata/locales/hr_HR
index 83087a3b02..d00e6bee25 100644
--- a/localedata/locales/hr_HR
+++ b/localedata/locales/hr_HR
@@ -24,8 +24,8 @@ tel        ""
 fax        ""
 language   "Croatian"
 territory  "Croatia"
-revision   "2.3"
-date       "2016-04-16"
+revision   "2.4"
+date       "2022-12-03"
 
 category "i18n:2012";LC_IDENTIFICATION
 category "i18n:2012";LC_CTYPE
@@ -171,8 +171,8 @@ nostr   "ne"
 END LC_MESSAGES
 
 LC_MONETARY
-int_curr_symbol           "HRK "
-currency_symbol           "kn"
+int_curr_symbol           "EUR "
+currency_symbol           "€"
 mon_decimal_point         ","
 mon_thousands_sep         "."
 mon_grouping              3;3
diff --git a/stdlib/tst-strfmon_l.c b/stdlib/tst-strfmon_l.c
index cc5da3e2ba..73ca0ea7b7 100644
--- a/stdlib/tst-strfmon_l.c
+++ b/stdlib/tst-strfmon_l.c
@@ -181,12 +181,12 @@ static const struct locale_pair tests[] =
       "hr_HR.UTF-8",
       {
         {
-          "HRK 1.234.567,89", "1.234.567,89 kn",
-          "HRK 1234567,89", "1234567,89 kn"
+          "EUR 1.234.567,89", "1.234.567,89 €",
+          "EUR 1234567,89", "1234567,89 €"
         },
         {
-          "-HRK 1.234.567,89", "-1.234.567,89 kn",
-          "-HRK 1234567,89", "-1234567,89 kn"
+          "-EUR 1.234.567,89", "-1.234.567,89 €",
+          "-EUR 1234567,89", "-1234567,89 €"
         }
       }
     },
